### ISSUE TITLE
multi submethod

### ISSUE THREAD

**Issue by ziuq** [Raku/doc#3605]
## Problem or new feature
https://docs.raku.org/language/objects

Just under the Methods header:

`Methods are declared with the method keyword inside a class body.`

Ignores submethods are subset.

The Method header , logically, should be over all the following headers through Submethods, except perhaps self.

https://docs.raku.org/language/objects#Submethods

Nowhere  in the docs are submethods and  multi  shown working together.

Suggestions
Submethods could be mentioned in the Multi docs.  An example would be good; one with BUILD might be better for two early issues illuminated at once.

**Comment by JJ** [comment]
True. We already know this #3063. Add more context would definitely be a good thing.
